src/themes/Default.jsx

```diff
@@ -11,7 +11,7 @@ const boxShadow =
 export const config = {
   boxShadow,
   font:
-    '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
+    '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif,FontAwesome',
```

Comment:
Why do you add FontAwesome here? It's the default fonts list.

Reply:
It was one of my many attempts to make the icon display...